.github/workflows/cypress.yml

```diff
@@ -0,0 +1,35 @@
+name: Cypress
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+env:
+  APP_NAME: nextcloud-vue
+
+jobs:
+  jest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ['12.x']
+
+    name: Runner ${{ matrix.containers }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up node ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cypress run
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+        run: npm run cypress:record
```

Comment:
I'm not fond, I think that complicates the setup a lot.
Why did we need to load this inside a docker container?

Reply:
> Why did we need to load this inside a docker container?

Because the tests now pass, and failed before 😉 

The tiny differences we saw with the font rendering are caused by running the tests on different machines. Between them, maybe the browser, the display settings, the available fonts or whatever else differs slightly, leading to tiny differences in the rendering which make the tests fail.

Of course, we could increase the difference threshold to make the tests pass. But this is not really an option as it would lead to false positives.

The difference in this image is 0.1623 and is not a regression (it's the difference between running locally on my machine and inside the docker container):
![AppSidebar vue - starred_null - compact_false - header_none - secondary_none-diff](https://user-images.githubusercontent.com/2496460/91408277-e6bfbe00-e843-11ea-93a8-7af1f4ae8068.png)

The difference in this image is 4 times lower with 0.0406 and it clearly is a regression (I intentionally changed the padding of the close button):
![AppSidebar vue - starred_null - compact_false - header_none - secondary_none-diff](https://user-images.githubusercontent.com/2496460/91408773-b2003680-e844-11ea-993d-9048d5597c7a.png)


So, running the visual tests in the very same environment ensured by a docker container seems like the only option to fix this. If you have another idea, I am open to check it out.

> I think that complicates the setup a lot.

It sure is complicated, but I got the impression, so is visual regression testing in general 🙈

Reply:
Cypress actually have dockerized github actions as well.
We could have use those?

I'm just not fond of having to also maintain a Docker setup there.

Reply:
https://github.com/cypress-io/github-action#docker-image

Reply:
I am fine with using this github action docker setup, but we have to as well locally run cypress within the container. How would you do that then?